src/Equations/DreicerNeuralNetwork.cpp

```diff
@@ -82,13 +84,48 @@ real_t DreicerNeuralNetwork::RunawayRate(
     real_t logNfree = log(nfree);
     real_t free_tot = nfree / ntot;
     real_t logTheta = log(T/Constants::mc2inEV);
+    
+    return nn_fix(fabs(E)/ED, logTheta, Zeff, Zeff0, Z0Z, Z0_Z,
+            logNfree, free_tot, nfree, tauEE, true);
+}
 
-    real_t rr = RunawayRate_derived_params(
-        fabs(E)/ED, logTheta, Zeff, Zeff0, Z0Z, Z0_Z,
-        logNfree, free_tot
-    );
-
-    return 4.0/(3.0*M_SQRTPI)*(nfree/tauEE) * rr;
+/**
+*Extrapolating the result from the NN such that it can be used for small E/ED(The
+*NN has not been trained for those values). Parameters are explained in the 
+*RunawayRate_derived_params function.
+*use_fix :  If enabled the return value will be an extrapolation of the form
+*exp(a * x^b)
+*/
+real_t DreicerNeuralNetwork::nn_fix(
+    const real_t EDD, const real_t logTheta, const real_t Zeff,
+    const real_t Zeff0, const real_t Z0Z, const real_t Z0_Z, const real_t logNfree, 
+    const real_t nfree_ntot, const real_t nfree, const real_t tauEE, bool use_fix
+) {
+    real_t boundary = 0.015;
+    real_t dE = 0.0001;
+    if(use_fix == true & EDD < boundary){
```

Comment:
In C/C++ you should use double ampersands ``&&`` for logical AND in if statements. The single ampersand is a bit-wise AND operation, which in this case yields the same result.